src/extension/sdk/dev_tools/embedded_view.ts

```diff
@@ -25,6 +26,9 @@ window.addEventListener('message', (event) => {
 			// https://github.com/flutter/devtools/issues/2775
 			window.dispatchEvent(new KeyboardEvent('keydown', message.data));
 			break;
+		case "launchUrl":
+			vscode.postMessage({command: 'launchUrl', data: message.data});
+			break;
```

Comment:
I wonder if we could call the post the same `did-click-link` message that VS Code seems to (to avoid having our own)?

https://github.com/microsoft/vscode/search?q=did-click-link

Reply:
Doesn't look like it to me; it would require calling `hostMessaging.postMessage('did-click-link', url)` but the [VSCode message method](https://github.com/microsoft/vscode/blob/99c8e788d46c935aecb16129919c6041eed92250/src/vs/workbench/contrib/webview/browser/pre/main.js#L947) requires the posted command to be "onmessage" or "do-update-state".